### PR TITLE
Adm NNS cmds

### PR DESCRIPTION
**Pull request by roman-khimov**
Enough to at least have some visibility into the NNS and be able to renew things.

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/dump_names.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	var res = make([]nameExp, 0)
+	for toks, err := tokIter.Next(10); len(toks) != 0 && err == nil; toks, err = tokIter.Next(10) {
+		for i := range toks {
+			var name = string(toks[i])
+			if zone != "" && name != zone && !strings.HasSuffix(name, "."+zone) {
+				continue
+			}
+			props, err := n11r.Properties(toks[i])
+			if err != nil {
+				cmd.PrintErrf("Error getting properties for %s: %v\n", string(toks[i]), err)
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/morph/dump_names.go:50]:
```suggestion
				cmd.PrintErrf("Error getting properties for %s: %v\n", toks[i], err)
```

**Review by cthulhu-rider — APPROVED** [review]

File: cmd/neofs-adm/internal/modules/morph/dump_names.go (hunk 1 above)
Comment (roman-khimov) [reply cmd/neofs-adm/internal/modules/morph/dump_names.go]: In fact, it can be `name`, fixed.

Comment (cthulhu-rider) [reply cmd/neofs-adm/internal/modules/morph/dump_names.go]: same below

Comment (roman-khimov) [reply cmd/neofs-adm/internal/modules/morph/dump_names.go]: Yep, fixed as well.

**Pull request merged**